sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -252,6 +253,13 @@ private CorePropertyDefinitions() {
         .defaultValue(Boolean.toString(false))
         .category(CATEGORY_ORGANIZATIONS)
         .type(PropertyType.BOOLEAN)
+        .build(),
+      PropertyDefinition.builder(ORGANIZATIONS_CREATE_PERSONAL_ORG)
```

Comment:
What about defining an internal property but not a hidden property ? That would allow to check that organization support is enabled.